libraries/bdp-access-proxy/bdp-access-impl/src/main/java/com/bosch/osmi/bdp/access/impl/BdpApiAccessImpl.java

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright (c) Bosch Software Innovations GmbH 2015.
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License Version 2.0 as published by the
+ * Free Software Foundation with classpath exception.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU General Public License version 2.0 for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program (please see the COPYING file); if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+ * 02110-1301, USA.
+ */
+
+package com.bosch.osmi.bdp.access.impl;
+
+import com.blackducksoftware.sdk.protex.client.util.ProtexServerProxy;
+import com.blackducksoftware.sdk.protex.client.util.ServerAuthenticationException;
+import com.blackducksoftware.sdk.protex.component.ComponentApi;
+import com.blackducksoftware.sdk.protex.license.LicenseApi;
+import com.blackducksoftware.sdk.protex.project.ProjectApi;
+import com.blackducksoftware.sdk.protex.project.bom.BomApi;
+import com.blackducksoftware.sdk.protex.project.codetree.CodeTreeApi;
+import com.blackducksoftware.sdk.protex.project.codetree.discovery.DiscoveryApi;
+import com.blackducksoftware.sdk.protex.user.UserApi;
+import com.bosch.osmi.bdp.access.api.BdpApiAccess;
+import com.bosch.osmi.bdp.access.api.model.User;
+import com.bosch.osmi.bdp.access.impl.model.UserImpl;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+
+/**
+ * Allows an access to the Bdp APIs. An access has to be initialized with a username and a password for which
```

Comment:
Really minor nitpicking: "an access" does not sound right to me.
Please feel absolutely free to ignore this comment.